monai/networks/blocks/denseblock.py

```diff
@@ -129,3 +129,177 @@ def _get_layer(self, in_channels, out_channels, dilation):
                 dilation=dilation,
                 bias=self.bias,
             )
+
+
+class ConvConcatDenseBlock(ConvDenseBlock):
```

Comment:
Similarly the classes added here should go with the Quicknat implementation if they aren't general purpose.